kong/plugins/prometheus/exporter.lua

```diff
@@ -1,11 +1,14 @@
+local balancer = require "kong.runloop.balancer"
```

Comment:
Yeah, I noticed the inconsistencies in this module but I opted for not addressing them to keep the PR simple. I think it might be a good idea to address those inconsistencies and style issues collectively in a separate PR -- which would be easier to review.